<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameControl3</class>
 <widget class="QFrame" name="FrameControl3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxShowCompass">
     <property name="text">
      <string>Show Compass</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxShowHM">
     <property name="text">
      <string>Show Heading Marker</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Orientation :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxMotion">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>N Up</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>H up</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
